flink-runtime/src/main/java/org/apache/flink/runtime/state/changelog/SequenceNumberRange.java

```diff
@@ -29,7 +29,7 @@ public interface SequenceNumberRange {
     long size();
 
     /**
-     * @return true if {@link #from} &lt; sqn &lt; {@link #to} (this implies that the range is not
+     * @return true if {@link #from} &le; sqn &lt; {@link #to} (this implies that the range is not
      *     empty, i.e. to &gt; from))
```

Comment:
minor: the number of right parenthesis should be just one.

Reply:
Thanks @masteryhx , you are right.